README.md

```diff
@@ -138,6 +138,21 @@ The B3 codec assumes it will receive lowercase HTTP headers, as this seems
 to be the standard in the popular frameworks like Flask and Django.
 Please make sure your framework does the same.
 
+## Prometheus metrics
+
+This module brings a [Prometheus](https://github.com/prometheus/client_python) integration to the internal Jaeger metrics.  
+The way to initialize the tracer with Prometheus metrics:  
+
+```python
+from jaeger_client.metrics_factory.prometheus_metrics import PrometheusMetricsFactory
+
+config = Config(
+        config={},
+        service_name='your-app-name',
+        metrics_factory=PrometheusMetricsFactory(namespace=service_name)
```

Comment:
maybe `namespace='your-app-name'` for consistency?

Reply:
add namespace='your-app-name'